server/index.js

```diff
@@ -20,10 +20,10 @@ app.post('/items/:item_id/seller', (req, res) => {
   let args = [seller_rating, total_sales, seller_name, seller_city, seller_state, on_etsy_since];
   seller.create(args)
   .then(() => {
-    res.status(201).end();
+    res.status(200).send('Successfully added new seller');
```

Comment:
I see you changed it back to 200. Maybe I won't change my error then :)

Reply:
I'm not sure its all THAT important since a simple OK will suffice here. I actually changed it back to 200 in order to more closely model how yours was set up.